wp-content/themes/pub/wporg-learn-2020/patterns/de-lesson-plan.php

```diff
@@ -0,0 +1,391 @@
+<h2>Anmerkungen für Lehrer</h2>
+<!-- /wp:heading -->
+
+<!-- wp:list -->
+<ul>
+<!-- wp:list-item -->
+<li>Notiz 1</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Notiz 2</li>
+<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:paragraph -->
+<p>Eine Liste von Tips und Notizen für den Moderator.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><em>Zum Beispiel:</em></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:list -->
+<ul>
+<!-- wp:list-item -->
+<li>Teilnehmende sollten das Twenty Twenty-Two Theme installiert haben</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Hinweise für den Fall das kein Projektor oder Internet zur Verfügung steht.</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Tips für den Fall das jemand nicht die nötigen Materialien zur Verfügung hat.</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Wie man mit verschiedenen Meinungen zu diesem Thema umgehen kann.</li>
+<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:heading {"lock":{"move":true,"remove":true}} -->
+<h2>Lektion Übersicht</h2>
+<!-- /wp:heading -->
+
+<!-- wp:list -->
+<ul>
+<!-- wp:list-item -->
+<li>Zuerst dies</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Dann das</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Zuletzt dies</li>
+<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:paragraph -->
+<p>Eine Gliederung der Lektion. </p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p><em>Zum Beispiel:</em></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:list -->
+<ul>
+<!-- wp:list-item -->
+<li>Erkläre was ein Theme ist</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Demonstriere wie ein Theme heruntergeladen und aktiviert wird.</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Lernende üben selber ein Theme herunterzuladen und zu installieren.</li>
+<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:heading {"lock":{"move":true,"remove":true}} -->
+<h2>Übungen</h2>
+<!-- /wp:heading -->
+
+<!-- wp:paragraph -->
+<p><strong>Name der Übung</strong></p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Kurze Beschreibung der Übung und welche Fähigkeiten sie vertiefen soll.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:list -->
+<ul>
+<!-- wp:list-item -->
+<li>Übung 1</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Übung 2</li>
+<!-- /wp:list-item -->
+</ul>
+<!-- /wp:list -->
+
+<!-- wp:quote -->
+<blockquote class="wp-block-quote">
+<!-- wp:paragraph -->
+<p>Dies sind kurze oder spezifische Aktivitäten, die den Teilnehmenden helfen, bestimmte Bestandteile der Lektion zu üben. Es sollten keine vollständig geskripteten Übungen sein, sondern etwas, das die Teilnehmer selbst durchführen können. Dies kann eine Übung, basierend auf einem Schritt der Lektion, sein.</p>
+<!-- /wp:paragraph -->
+</blockquote>
+<!-- /wp:quote -->
+
+<!-- wp:heading {"lock":{"move":true,"remove":true}} -->
+<h2>Quiz</h2>
+<!-- /wp:heading -->
```

Comment:
Sure that this is a „Quiz“? Not Quiz => Bewertung (oder Beurteilung)?

Reply:
Ich denke Quiz passt gut. Es ist ein kurzer Test, der gelerntes überprüft.